src/dashboard/components/button.py

```diff
@@ -0,0 +1,38 @@
+"""Button component."""
+from typing import Dict
+
+from dash import html
+
+from dashboard.components.icon import icon
+
+
+def button(
```

Comment:
Ser bra ut, skulle dock vara nice om man kan lägga in en egen className arg också för custom styling